osmosis/assetlist.json

```diff
@@ -1169,6 +1169,53 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.svg"
       }
+    },
+    {
+      "description": "AUTISM on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/9DDF52A334F92BC57A9E0D59DFF9984EAC61D2A14E5162605DF601AA58FDFC6D",
+          "exponent": 0,
+          "aliases": [
+            "uautism"
+          ]
+        },
+        {
+          "denom": "autism",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/9DDF52A334F92BC57A9E0D59DFF9984EAC61D2A14E5162605DF601AA58FDFC6D",
+      "name": "Injective AUTISM",
+      "display": "autism",
+      "symbol": "AUTISM",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism",
+            "channel_id": "channel-122"
+          },
+          "chain": {
+            "channel_id": "channel-8",
+            "path": "transfer/channel-8/factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "injective",
+            "base_denom": "utia"
```

Comment:
This doesn't seem right?

Reply:
@JeremyParish69 can you advise? I'm bad at these.

Reply:
It shouldn't be trying to synchronize to the TIA image. It should be pointing to the original AUTISM image. So base_denom is wrong here

Reply:
I'll fix shortly.. does everything else  look ok?
I haven't done a token factory one before

Reply:
Other than my other comment, I think everything else is fine